src/topology/path_connected.lean

```diff
@@ -465,11 +465,9 @@ end
   mean the uncurried function which maps `(t₀, t₁, s)` to `γ.truncate t₀ t₁ s` is continuous. -/
 lemma truncate_continuous_family {X : Type*} [topological_space X] {a b : X}
   (γ : path a b) : continuous (λ x, γ.truncate x.1 x.2.1 x.2.2 : ℝ × ℝ × I → X) :=
-begin
-  simp only [has_coe_to_fun.coe, coe_fn, path.truncate],
-  continuity,
-  exact continuous_subtype_coe
-end
+(γ.continuous.comp continuous_proj_I).comp
+  (((continuous_subtype_coe.comp (continuous_snd.comp continuous_snd)).max continuous_fst).min
+    (continuous_fst.comp continuous_snd))
```

Comment:
I would maybe keep the old proof in a comment as a TODO for when `continuity` becomes more efficient.